templates/src/diagnostic.c.erb

```diff
@@ -86,6 +86,7 @@ static const pm_diagnostic_data_t diagnostic_messages[PM_DIAGNOSTIC_ID_MAX] = {
 
     // Errors that should raise syntax errors
     [PM_ERR_ALIAS_ARGUMENT]                     = { "invalid argument being passed to `alias`; expected a bare word, symbol, constant, or global variable", PM_ERROR_LEVEL_SYNTAX },
+    [PM_ERR_ALIAS_ARGUMENT_GLOBAL_VARIABLE]     = { "invalid argument being passed to `alias`; when aliasing global variables, both the old and new names must be global variables", PM_ERROR_LEVEL_SYNTAX },
```

Comment:
I think the message can be more specific. Since this error only happens when the first (new) name is already a global variable, we can probably say 

```
invalid argument being passed to `alias`; expected the old name to be a global variable as well
```

Reply:
`old name` might be unclear if it's the first or second argument (`alias` is notoriously confusing about this), so I'd suggest to use `the second name` instead of `the old name`, or the message in the current state of this PR (which sounds clear to me).